openedx_proversity_reports/middleware.py

```diff
@@ -0,0 +1,98 @@
+"""
```

Comment:
How the calculation will work, if i have two or more tabs open? @andrey-canon

Reply:
in the same way